keystore/src/entities/mod.rs

```diff
@@ -172,6 +178,17 @@ cfg_if::cfg_if! {
             id: Vec<u8>,
         }
 
+        #[async_trait::async_trait(?Send)]
+        pub trait EntityMlsExt: Entity {
+            async fn mls_delete<'a>(tx: &crate::connection::storage::WasmStorageTransaction<'a>, id: StringEntityId<'a>) -> CryptoKeystoreResult<()> {
+                tx.delete(Self::COLLECTION_NAME, id.as_slice()).await
+            }
+            async fn mls_save<'a>(&'a self, tx: &crate::connection::storage::WasmStorageTransaction<'a>) -> CryptoKeystoreResult<()>;
+            async fn mls_insert<'a>(&'a self, _tx: &crate::connection::storage::WasmStorageTransaction<'a>) -> CryptoKeystoreResult<Self::AutoGeneratedFields> {
+                return Err(CryptoKeystoreError::NotImplemented);
```

Comment:
```suggestion
                Err(CryptoKeystoreError::NotImplemented)
```

Reply:
I've removed this function entirely later since there's no one using it and the few that were, don't fit in the transaction model